clippy_lints/src/utils/visitors.rs

```diff
@@ -0,0 +1,125 @@
+use rustc_hir as hir;
```

Comment:
Introducing a visitor module for all our (shared) visitors is actually a really good idea! But this is something for another PR.

Reply:
Okay, I'm reverting c4e24c8 and will make another PR containing the commit after this PR gets merged. Does this sounds good?

Reply:
Nah, don't revert the commit, we can do this here. The extra PR for the **other** visitors would still be appreciated though!

Reply:
That's a perfect task for my spare-time activities :)